src/main/java/org/folio/rest/impl/EholdingsPackagesImpl.java

```diff
@@ -260,7 +264,11 @@ public void getEholdingsPackagesResourcesByPackageId(String packageId, String so
     if (tags == null){
       return CompletableFuture.completedFuture(new PackageResult(packageId, null, null));
     }else {
-      return tagRepository.updateRecordTags(tenant, packageId.getVendorId() + "-" + packageId.getPackageId(), RecordType.PACKAGE, tags.getTagList())
+      return
+        packageRepository.savePackage(packageId, tenant)
```

Comment:
Can we rename packageId to packageData or packageByIdData instead because that is really what it is?

Reply:
Thanks, fixed